etcd-v2/docker-compose.yml

```diff
@@ -0,0 +1,98 @@
+version: '2'
```

Comment:
This version is of the docker-compose spec not etcd.

Reply:
Good catch. Thanks

Reply:
lets also change the path as this is v3?

Reply:
I let the path etcd-v2 so we could define an etcd-v3 for docker-compose version 3. It only allows 1 compose file per directory? 

Suggesting to new file directory name?